src/netlify/deploy/README.md

```diff
@@ -0,0 +1,52 @@
+# `garden netlify-deploy`
+
+Deploy the given directory to a [Netlify](https://www.netlify.com/) site.
```

Comment:
In general, yes – it eases the means by which we will CI-publish PRs for review. However, it can also be run as a local command to deploy content to a Netlify site. Both have value.